src/NuGet.Server/Infrastructure/ServerPackageRepository.cs

```diff
@@ -36,7 +36,8 @@ public class ServerPackageRepository
 
         private readonly bool _runBackgroundTasks;
         private FileSystemWatcher _fileSystemWatcher;
-        private bool _isFileSystemWatcherSuppressed;
+        private bool _isFileSystemWatcherSuppressed = false;
```

Comment:
Obviously not necessary, but there is a decent amount of state/bookkeeping in this class so I thought it would be clearer.